src/plugins/event-plugins/__tests__/XhrPlugin.test.ts

```diff
@@ -609,7 +618,7 @@ describe('XhrPlugin tests', () => {
         expect(record).toHaveBeenCalledTimes(1);
     });
 
-    test('when recordAllRequests is false then the plugin does record a request with status OK', async () => {
+    test('when recordAllRequests is true then the plugin records a request with status OK', async () => {
```

Comment:
suggestion: It appears that we only updated the test name without changing the test content. Do we want to update the test to add validations?

Reply:
oh good catch, i think it accidentally got lost when resolving merge conflicts

Reply:
On second look, I believe this change was to fix a typo. `recordAllRequests` is true in the configuration but was described to be false in the test name